### PR TITLE
Tn en astronomical no

### PR DESCRIPTION
**Pull request by anand-nv**
# What does this PR do ?

Adds support for normalization of numbers larger than a trillion

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [X] Did you write any new necessary tests?
- [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [X] New Feature

### CODE REVIEW

File: nemo_text_processing/text_normalization/en/taggers/cardinal.py [hunk 1]
```diff
@@ -44,6 +44,7 @@ def __init__(self, deterministic: bool = True, lm: bool = False):
         self.deterministic = deterministic
         # TODO replace to have "oh" as a default for "0"
         graph = pynini.Far(get_abs_path("data/number/cardinal_number_name.far")).get_fst()
+        graph_au = pynini.Far(get_abs_path("data/number/cardinal_number_name_au.far")).get_fst()
```
Comment (yzhang123) [inline nemo_text_processing/text_normalization/en/taggers/cardinal.py:47]: the far file is not added

Comment (anand-nv) [reply nemo_text_processing/text_normalization/en/taggers/cardinal.py:47]: It's been added. https://github.com/NVIDIA/NeMo-text-processing/blob/5629512439662a1b7dbb359315c877ad63c6f363/nemo_text_processing/text_normalization/en/data/number/cardinal_number_name_au.far

**Review by yzhang123 — DISMISSED** [review]

**Comment by jimregan** [comment]
> 124,444,234,854,823,834,553~one hundred twenty four quintillion four hundred forty four quadrillion two hundred thirty four trillion eight hundred fifty four billion eight hundred twenty three million eight hundred thirty four thousand five hundred and fifty three

Um... As a speaker of non-US English, I find this a strange mix of US/non-US number patterns. "one hundred and twenty four quintillion four hundred and forty four quadrillion two hundred and thirty four trillion eight hundred and fifty four billion eight hundred and twenty three million eight hundred and thirty four thousand five hundred and fifty three" is how it would typically be read on this side of the Atlantic.

**Comment by yzhang123** [comment]
@anand-nv could you take a look ^

**Comment by anand-nv** [comment]
https://english.stackexchange.com/questions/3518/american-vs-british-english-meaning-of-one-hundred-and-fifty.

**Pull request merged**